.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Any reason for this?

Reply:
it happened when husky was added - i didn't add it explicitly.

/me does his own research

Looks like husky auto-gens it when installing: https://github.com/typicode/husky/issues/929